src/main.cpp

```diff
@@ -1,97 +1,58 @@
 #include <Arduino.h>
 #include <SPI.h>
 
-#ifdef ESP32
-#include <WiFiManager.h>
+#ifdef ESP8266
+#include <ESP8266WiFi.h>
 #endif
 #ifdef ESP32
+#include <WiFi.h>
+#include <IPAddress.h>
 #include <ESPmDNS.h>
 #endif
-#ifdef ESP8266
-#include <ESP8266WiFi.h>
-#endif
 
 #include "PluginManager.h"
 
-#include "plugins/BreakoutPlugin.h"
-#include "plugins/CirclePlugin.h"
 #include "plugins/DrawPlugin.h"
-#include "plugins/FireworkPlugin.h"
+#include "plugins/BreakoutPlugin.h"
+#include "plugins/SnakePlugin.h"
 #include "plugins/GameOfLifePlugin.h"
+#include "plugins/StarsPlugin.h"
 #include "plugins/LinesPlugin.h"
+#include "plugins/CirclePlugin.h"
 #include "plugins/RainPlugin.h"
-#include "plugins/SnakePlugin.h"
-#include "plugins/StarsPlugin.h"
+#include "plugins/FireworkPlugin.h"
 
 #ifdef ENABLE_SERVER
-#include "plugins/AnimationPlugin.h"
 #include "plugins/BigClockPlugin.h"
 #include "plugins/ClockPlugin.h"
 #include "plugins/WeatherPlugin.h"
+#include "plugins/AnimationPlugin.h"
+#include "mqtt.h"
 #endif
 
-#include "asyncwebserver.h"
+#include "websocket.h"
+#include "secrets.h"
 #include "ota.h"
+#include "webserver.h"
```

Comment:
It's not in the project?

Reply:
Typo, this should be asyncwebserver.h as in the current master - fixed.